seriously-simple-podcasting.php

```diff
@@ -89,4 +89,4 @@
 
 if ( ssp_is_elementor_ok() ) {
 	$elementor_widgets = new Elementor_Widgets();
```

Comment:
I am actually ok with leaving the elementor integration as one class file, that manages the widgets and the importer. They are all part of the same functionality, so let's leave it as is.